app/forms/ALFForms.scala

```diff
@@ -152,19 +153,19 @@ object ALFForms extends EmptyStringValidator {
   def ukEditForm(optConfig: Option[ManualAddressEntryConfig] = None)(implicit messages: Messages): Form[Edit] = {
     val config = optConfig getOrElse ManualAddressEntryConfig()
     Form(
-    mapping(
-      "organisation" -> optional(text),
-      "line1" -> atLeastOneAddressLineOrTown(messages(s"constants.editPageAtLeastOneLineOrTown")).verifying(constraintOptStringMaxLength(messages(s"constants.editPageAddressLine1MaxErrorMessage", config.line1MaxLength + 1), config.line1MaxLength)),
-      "line2" -> optional(text).verifying(constraintOptStringMaxLength(messages(s"constants.editPageAddressLine2MaxErrorMessage", config.line2MaxLength + 1), config.line2MaxLength)),
-      "line3" -> optional(text).verifying(constraintOptStringMaxLength(messages(s"constants.editPageAddressLine3MaxErrorMessage", config.line3MaxLength + 1), config.line3MaxLength)),
-      "town" -> optional(text).verifying(constraintOptStringMaxLength(messages(s"constants.editPageTownMaxErrorMessage", config.townMaxLength + 1), config.townMaxLength)),
-      "postcode" -> default(text, ""),
-      "countryCode" -> ignored[String]("GB")
-    )(Edit.apply)(Edit.unapply)
-  )
+      mapping(
+        "organisation" -> optional(text),
+        "line1" -> atLeastOneAddressLineOrTown(messages(s"constants.editPageAtLeastOneLineOrTown")).verifying(constraintOptStringMaxLength(messages(s"constants.editPageAddressLine1MaxErrorMessage", config.line1MaxLength + 1), config.line1MaxLength)),
+        "line2" -> optional(text).verifying(constraintOptStringMaxLength(messages(s"constants.editPageAddressLine2MaxErrorMessage", config.line2MaxLength + 1), config.line2MaxLength)),
+        "line3" -> optional(text).verifying(constraintOptStringMaxLength(messages(s"constants.editPageAddressLine3MaxErrorMessage", config.line3MaxLength + 1), config.line3MaxLength)),
+        "town" -> optional(text).verifying(constraintOptStringMaxLength(messages(s"constants.editPageTownMaxErrorMessage", config.townMaxLength + 1), config.townMaxLength)),
+        "postcode" -> default(text, ""),
+        "countryCode" -> ignored[String]("GB")
+      )(Edit.apply)(Edit.unapply)
+    )
```

Comment:
Does this form also need the new validation checks added against it? Or is the idea that these two seaprate forms should be removed now and the unified new "editForm" be used everywhere?